### DISCUSSION TITLE
How do I specify where the configuration file should be created/named ?

### DISCUSSION THREAD

**Discussion by azro352** [renovatebot/renovate#24241] [Request Help] [answered]
Hi

The documentation states that renovate will use look for [various names][0] for the configuration file but can we specify the default location for the onboarding PR ? 

I want renovate to create the file at `.gitlab/renovate.json`

I searched in the doc, without success

Thanks

> Self-hosted  [36.79.1-slim][1] gitlab.com

[0]: https://docs.renovatebot.com/configuration-options/
[1]: https://hub.docker.com/layers/renovate/renovate/36.79.1-slim/images/sha256-95f0cd072fce311127f25b4d7c48c3922afbb074554aaac4d06559ee18d35ce7?context=explore

**Comment by rarkins** [comment, marked as answer]
https://docs.renovatebot.com/self-hosted-configuration/

onboardingConfigFileName
